CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+Contributions are accepted as pull requests.  Please observe our coding
+practices at https://github.com/City-of-Helsinki/bestpractice/ .
+Please make your pull requests short, elegant and only handling one
+issue at a time!
+
+If you make a pull request, you may also want to contact
+<INSERT SERVICE'S ROLE ADDRESS> to tell about your contribution.
```

Comment:
I wonder if there's an email we could use here. :thinking:

Reply:
I think there is a [gitter chat room](https://gitter.im/City-of-Helsinki/heldev), we could mention it here. OR just delete this paragraph 🤷

Reply:
so, what do you think we should do here?

Reply:
I'll remove it for now. I shall add it back when I figure out the proper thing to put there.